dev.neovide.neovide.metainfo.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>dev.neovide.neovide</id>
+  <name>neovide</name>
+  <summary>No Nonsense Neovim Client in Rust</summary>
```

Comment:
If you can come up with a better summary https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/quality-guidelines#not-technical